### ISSUE TITLE
Internal server error...

### ISSUE THREAD

**Issue by brdobrdo** [DJaeger/files_external_dropbox#15]
Just installed from Nextcloud and got Internal server error. And what now...
Nextcloud 18 on shared hosting.

Renamed files_external
And files_external_dropbox
Now I can open nextcloud from browser but i cannot remove dropbox app, and other app for external files is also disabled... What to do?

**Comment by dstrifler** [comment]
Same as above.  Junk plugin just ruined my Nextcloud.

Was getting internal server errors and had to disable this plugin as well as Nextcloud's own native external storage plugin to regain access.

Not going to be a very happy camper if I have to jump through a bunch of hoops to move around gigs of existing data on my server and re-install a fresh Nextcloud just to restore the native external storage due to a half baked dropbox plugin.

**Comment by brdobrdo** [comment]
Ok... Found a solution. If you have external storage support installed and then install external dropbox support. The thing brakes somewhere. There is a line in app.php of dropbox package saying something about enabling files_external.
My solution was to rename files_external, leave the Dropbox folder as it is in cpanel and then opened nextcloud from web browser, uninstalled dropbox from there, back in cpanel to rename files_external to original name.

**Comment by panpanika** [comment]
Hi, I got the same issues as the people above me. (I repaired via rolling back snapshot), from the log, this caught my eyes:
Base table or view not found: 1146 Table 'nextcloud.external_mounts' doesn't exist

hope this helps to sort this out somehow, I would love to use this extension..
thanks

**Comment by pablomenino** [comment]
I have the same issue, remember that you can disable extensions from command line.

**List extensions**

```
cd /PATH/TO/NEXTCLOUD
sudo -u www-data php occ app:list
```

**disable files_external_dropbox extension**

```
cd /PATH/TO/NEXTCLOUD
sudo -u www-data php occ app:disable files_external_dropbox
```

**Comment by ismailyenigul** [comment]
Hello
I explained the root cause of the problem at https://github.com/nextcloud/server/issues/21025
To fix the problem you need to disable `files_external_dropbox and files_external`
and enable files_external on nextcloud ui manually

## how to disable apps in docker env
```
docker exec -it nextcloud_container_id /bin/bash
chsh -s /bin/bash www-data  # because there  is no sudo in docker image
su - www-data
php /var/www/html/occ   app:disable files_external
php /var/www/html/occ   app:disable files_external_dropbox
```

Then Go to UI->Apps
First, enable File External Storage then Dropbox

**Comment by pablomenino** [comment]
After installing and configure files_external, I was able to install files_external_dropbox., Thanks.

In my case the problem was installing both apps in the same time.

**Comment by DJaeger** [comment]
Fixed by 469ab4d28cc648bc210d081f8032ae26d3f0da05
